yaml/config.go

```diff
@@ -62,27 +71,50 @@ func (f *configFile) load(bytes []byte) error {
 	return nil
 }
 
-// MasterKeyStringsForFile returns a comma separated string of KMS ARNs and a comma separated list of PGP fingerprints. If the config bytes are left empty, the function will look for the config file by itself.
-func MasterKeyStringsForFile(filepath string, confBytes []byte) (kms, pgp string, err error) {
+func KeyGroupsForFile(filepath string, confBytes []byte, kmsEncryptionContext map[string]*string) ([]sops.KeyGroup, error) {
```

Comment:
Instead of returning a bunch of strings, the configuration now returns key groups instead.

Reply:
See my previous comment on the structure of the keygroup configuration